src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix3x2.cs

```diff
@@ -52,19 +59,22 @@ public struct Matrix3x2 : IEquatable<Matrix3x2>
             M32 = m32;
         }
 
-        /// <summary>Returns the multiplicative identity matrix.</summary>
+        /// <summary>Gets the multiplicative identity matrix.</summary>
+        /// <value>The multiplicative identify matrix.</value>
```

Comment:
What is this <value> tag? Is it new? Don't we use <return> here instead usually?

Reply:
Properties need their value to be described. Here's the full explanation of this tag in the dotnet-api-docs Wiki:

https://github.com/dotnet/dotnet-api-docs/wiki/Property-Value